packages/cudos-test/_run-single-test.sh

```diff
@@ -0,0 +1,46 @@
+source ./packages/cudos-test/_vars.sh
```

Comment:
Files `_vars.sh` `_run-single-test.sh` `_run-tests.sh` have a different purpose from the rest of the files in `cudos-test` folder. So how about putting 'tests' in a subfolder while 'test runners' are in `cudos-test` (or vice versa) instead of underscore in front of 'test runners' file names?